pages/Flask-of-Knowledge.md

```diff
@@ -0,0 +1,6 @@
+The Flask of Knowledge is part of the [Magical Gadgets](https://github.com/Slimefun/Slimefun4/wiki/Magical-Gadgets) category.
+
+This item can be used to store 1 level of experience into it and you will be able to collect that level at any point. It is also used as a crafting material in some recipes.
```

Comment:
It's not exactly fair to say you can collect the level imo, it only works as a normal exp bottle no matter what level you filled it at and certainly won't give you back a full level anyway. Maybe something like this?
```suggestion
This item can be used to store 1 level of experience into it and you will be able to collect some back at any point. It is also used as a crafting material in some recipes.
```

Reply:
(I know it's a bit picky sorry ahahaha)

Reply:
```suggestion
This item can be used to store 1 level of experience into it and it will become a Bottle o' Enchanting which you can use to regain some exp. It is also used as a crafting material in some recipes.
```